### ISSUE TITLE
Creating a new project without ActionCable generates cable.coffee file

### ISSUE THREAD

**Issue by sprobst** [rails/rails#22959]
I created a new Project without ActionCable like so:
`rails new rails5-backend --skip-action-cable`

After that, there is a file named `cable.coffee` under `app/assets/javascripts`, wich is included in `application.js`.

When I then scaffold a model, view and controller, opening one of the generated sites does not work, as `#= require action_cable` in `cable.coffee` can not be found.

Removing that file solves the problem.

For reference, I'm using `ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin15]`

**Comment by prathamesh-sonpatki** [comment]
@sprobst The switch to skip is `--skip-actioncable`. It will not create any files related to actioncable.

**Comment by sprobst** [comment]
@prathamesh-sonpatki Thank you! Then the output of `rails help`is wrong, as I copied it from there:

```
macbook:Fun simon$ rails help new
Usage:
  rails new APP_PATH [options]

Options:
  -r, [--ruby=PATH]                                      # Path to the Ruby binary of your choice
                                                         # Default: /Users/simon/.rbenv/versions/2.3.0/bin/ruby
  -m, [--template=TEMPLATE]                              # Path to some application template (can be a filesystem path or URL)
  -d, [--database=DATABASE]                              # Preconfigure for selected database (options: mysql/oracle/postgresql/sqlite3/frontbase/ibm_db/sqlserver/jdbcmysql/jdbcsqlite3/jdbcpostgresql/jdbc)
                                                         # Default: sqlite3
  -j, [--javascript=JAVASCRIPT]                          # Preconfigure for selected JavaScript library
                                                         # Default: jquery
      [--skip-gemfile], [--no-skip-gemfile]              # Don't create a Gemfile
  -B, [--skip-bundle], [--no-skip-bundle]                # Don't run bundle install
  -G, [--skip-git], [--no-skip-git]                      # Skip .gitignore file
      [--skip-keeps], [--no-skip-keeps]                  # Skip source control .keep files
  -M, [--skip-action-mailer], [--no-skip-action-mailer]  # Skip Action Mailer files
  -O, [--skip-active-record], [--no-skip-active-record]  # Skip Active Record files
  -C, [--skip-action-cable], [--no-skip-action-cable]    # Skip Action Cable files
  -S, [--skip-sprockets], [--no-skip-sprockets]          # Skip Sprockets files
      [--skip-spring], [--no-skip-spring]                # Don't install Spring application preloader
  -J, [--skip-javascript], [--no-skip-javascript]        # Skip JavaScript files
      [--skip-turbolinks], [--no-skip-turbolinks]        # Skip turbolinks gem
  -T, [--skip-test], [--no-skip-test]                    # Skip test files
      [--dev], [--no-dev]                                # Setup the application with Gemfile pointing to your Rails checkout
      [--edge], [--no-edge]                              # Setup the application with Gemfile pointing to Rails repository
      [--rc=RC]                                          # Path to file containing extra configuration options for rails command
      [--no-rc], [--no-no-rc]                            # Skip loading of extra configuration options from .railsrc file
      [--api], [--no-api]                                # Preconfigure smaller stack for API only apps

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist

Rails options:
  -h, [--help], [--no-help]        # Show this help message and quit
  -v, [--version], [--no-version]  # Show Rails version number and quit

Description:
    The 'rails new' command creates a new Rails application with a default
    directory structure and configuration at the path you specify.

    You can specify extra command-line arguments to be used every time
    'rails new' runs in the .railsrc configuration file in your home directory.

    Note that the arguments specified in the .railsrc file don't affect the
    defaults values shown above in this help message.

Example:
    rails new ~/Code/Ruby/weblog

    This generates a skeletal Rails installation in ~/Code/Ruby/weblog.
```

Using `Rails 5.0.0.beta1`

**Comment by prathamesh-sonpatki** [comment]
@sprobst My bad, the change to skip action cable files correctly was added in https://github.com/rails/rails/commit/96093e3cf79c0e809e13b02d12e0705f7acfdf31, it is not present in beta. **`--skip-action-cable`** is correct. On master branch the action cable related files are not created if we use it.

**Comment by pixeltrix** [comment]
@sprobst as @prathamesh-sonpatki says this is fixed on master and is a duplicate of #22869 - thanks for your report.
